hack/test-integration.sh

```diff
@@ -55,10 +55,14 @@ testexec="${testdir}/${name}.test"
 mkdir -p "${testdir}"
 
 # build the test executable (cgo must be disabled to have the symbol table available)
-pushd "${testdir}" &>/dev/null
-echo "Building test executable..."
-CGO_ENABLED=0 go test -c -tags="${tags}" "${OS_GO_PACKAGE}/${package}"
-popd &>/dev/null
+if [[ -n "${OPENSHIFT_SKIP_BUILD:-}" ]]; then
```

Comment:
Can do.

The dind deployment script uses OPENSHIFT_SKIP_BUILD to indicate that the deployment should take place without building the image or openshift, and the extended networking test runner uses the same flag to both skip building the test and control deployment.  I think the name should remain for the dind script, but would you like to see the test runner use TEST_ONLY for consistency?